scene/3d/cpu_particles_3d.cpp

```diff
@@ -881,53 +881,53 @@ void CPUParticles3D::_particles_process(double p_delta) {
 			p.custom[1] = p.time / lifetime;
 			tv = p.time / p.lifetime;
 
-			real_t tex_linear_velocity = 0.0;
+			real_t tex_linear_velocity = 1.0;
```

Comment:
I'm not sure about all these changes. The default settings for a 3D particle IMO should not be to have a fixed linear, orbital, angular velocity + accelerations.

It should be to have no start parameters of its own, and just be subjected to gravity. That's how it works in 2D too.

This is the current behavior (GPU on left, CPU on right):
https://user-images.githubusercontent.com/4701338/143868904-9e74b2ed-0831-4831-83b4-7b28ff76e901.mp4

If I understand this PR correctly (didn't compile to test), this would become:
https://user-images.githubusercontent.com/4701338/143869157-f9607ae4-09fb-43b0-8769-30e41328d784.mp4